src/utils/webrtc/models/CallParticipantCollection.js

```diff
@@ -15,36 +15,35 @@ export default function CallParticipantCollection() {
 
 	this._superEmitterMixin()
 
-	// FIXME: use reactive instead of ref after migration to vue 3
-	this.callParticipantModels = ref([])
+	this.callParticipantModels = reactive([])
```

Comment:
Did it cause a problem anywhere or just refactoring?

Reply:
I thought it could be an initial problem, but as we're touching it anyway, just refactored

Reply:
> Also `reactive` works for nested objects, and `ref` doesn't, right?

No, it does. `ref` creates a fully reactive variable, and for objects, it returns value as `reactive(_value)`. In general, there is nothing `ref` cannot do what we can with `reactive` and `reactive` is only better if we want to avoid writhing `.value`.